envi/cli.py

```diff
@@ -662,16 +663,16 @@ def do_search(self, line):
             return self.do_help('search')
 
         if options.is_expr:
-            import struct #FIXME see below
+            import struct  #FIXME see below
             sval = self.parseExpression(pattern)
-            pattern = struct.pack('<L', sval) # FIXME 64bit (and alt arch)
+            pattern = struct.pack('<L', sval)  # FIXME 64bit (and alt arch)
```

Comment:
here's where you "fixing" is gonna cost you ;)
could you please fix the fixme?  use `e_bits.buildbytes()` or `e_bits.getFormat()` with the correct arch psize and endianness :)